### PR TITLE
STYLE upgrade black formatter

### PR DESCRIPTION
**Pull request by MarcoGorelli**
gonna have to do this eventually...

I'm trying to tell if it changed anything other than just deleting newlines, any `git` masters know how to do that?

### CODE REVIEW

File: pandas/_libs/hashtable.pyi [hunk 1]
```diff
@@ -185,10 +185,13 @@ class HashTable:
...
-        np.ndarray,  # np.ndarray[subclass-specific]
-        npt.NDArray[np.intp],
-    ] | np.ndarray: ...  # np.ndarray[subclass-specific]
+    ) -> (
+        tuple[
+            np.ndarray,  # np.ndarray[subclass-specific]
+            npt.NDArray[np.intp],
+        ]
+        | np.ndarray
+    ): ...  # np.ndarray[subclass-specific]
```
Comment (rhshadrach) [inline pandas/_libs/hashtable.pyi:194]: Here

File: pandas/conftest.py [hunk 2]
```diff
@@ -168,7 +168,7 @@ def pytest_collection_modifyitems(items, config) -> None:
         if "/frame/" in item.nodeid:
             item.add_marker(pytest.mark.arraymanager)
 
-        for (mark, kwd, skip_if_found, arg_name) in marks:
+        for mark, kwd, skip_if_found, arg_name in marks:
```
Comment (rhshadrach) [inline pandas/conftest.py:171]: Here

File: pandas/core/algorithms.py [hunk 3]
```diff
@@ -1700,6 +1695,7 @@ def diff(arr, n: int, axis: AxisInt = 0):
 # --------------------------------------------------------------------
 # Helper functions
 
+
```
Comment (rhshadrach) [inline pandas/core/algorithms.py:1698]: Here

File: pandas/core/apply.py [hunk 4]
```diff
@@ -939,7 +937,7 @@ def series_generator(self):
                 yield obj._ixs(i, axis=0)
 
         else:
-            for (arr, name) in zip(values, self.index):
+            for arr, name in zip(values, self.index):
```
Comment (rhshadrach) [inline pandas/core/apply.py:940]: Here

File: pandas/core/dtypes/concat.py [hunk 5]
```diff
@@ -48,6 +48,7 @@ def concat_compat(to_concat, axis: AxisInt = 0, ea_compat_axis: bool = False):
     -------
     a single array, preserving the combined dtypes
     """
+
```
Comment (rhshadrach) [inline pandas/core/dtypes/concat.py:51]: Here

File: pandas/core/reshape/encoding.py [hunk 6]
```diff
@@ -199,7 +198,7 @@ def check_len(item, name):
             # columns to prepend to result.
             with_dummies = [data.select_dtypes(exclude=dtypes_to_encode)]
 
-        for (col, pre, sep) in zip(data_to_encode.items(), prefix, prefix_sep):
+        for col, pre, sep in zip(data_to_encode.items(), prefix, prefix_sep):
```
Comment (rhshadrach) [inline pandas/core/reshape/encoding.py:201]: Here

File: pandas/core/series.py [hunk 7]
```diff
@@ -235,6 +235,7 @@ def wrapper(self):
 # ----------------------------------------------------------------------
 # Series class
 
+
```
Comment (rhshadrach) [inline pandas/core/series.py:238]: Here

File: pandas/io/formats/style_render.py [hunk 8]
```diff
@@ -2152,7 +2151,7 @@ def _parse_latex_cell_styles(
     """
     if convert_css:
         latex_styles = _parse_latex_css_conversion(latex_styles)
-    for (command, options) in latex_styles[::-1]:  # in reverse for most recent style
+    for command, options in latex_styles[::-1]:  # in reverse for most recent style
```
Comment (rhshadrach) [inline pandas/io/formats/style_render.py:2154]: Here

File: pandas/io/formats/style_render.py [hunk 9]
```diff
@@ -2295,7 +2294,7 @@ def color(value, user_arg, command, comm_arg):
     }
 
     latex_styles: CSSList = []
-    for (attribute, value) in styles:
+    for attribute, value in styles:
```
Comment (rhshadrach) [inline pandas/io/formats/style_render.py:2297]: Here

File: pandas/plotting/_matplotlib/core.py [hunk 10]
```diff
@@ -718,7 +715,7 @@ def _adorn_subplots(self):
                             f"number of columns = {self.nseries}"
                         )
 
-                    for (ax, title) in zip(self.axes, self.title):
+                    for ax, title in zip(self.axes, self.title):
```
Comment (rhshadrach) [inline pandas/plotting/_matplotlib/core.py:718]: Here

File: pandas/tests/arithmetic/conftest.py [hunk 11]
```diff
@@ -20,6 +20,7 @@ def switch_numexpr_min_elements(request):
 
 # ------------------------------------------------------------------
 
+
```
Comment (rhshadrach) [inline pandas/tests/arithmetic/conftest.py:23]: Here

File: pandas/tests/arrays/categorical/test_indexing.py [hunk 12]
```diff
@@ -367,6 +364,7 @@ def non_coercible_categorical(monkeypatch):
     ValueError
         When Categorical.__array__ is called.
     """
+
```
Comment (rhshadrach) [inline pandas/tests/arrays/categorical/test_indexing.py:367]: Here

File: pandas/tests/generic/test_duplicate_labels.py [hunk 13]
```diff
@@ -177,7 +177,11 @@ def test_ndframe_getitem_caching_issue(self, request, using_copy_on_write):
                     pd.DataFrame({"A": [1, 2]}, index=["a", "b"]).set_flags(
                         allows_duplicate_labels=False
                     ),
-                    pd.Series([1, 2], index=["a", "b"], name="B",).set_flags(
+                    pd.Series(
+                        [1, 2],
+                        index=["a", "b"],
+                        name="B",
+                    ).set_flags(
```
Comment (rhshadrach) [inline pandas/tests/generic/test_duplicate_labels.py:184]: Here

File: pandas/tests/generic/test_label_or_level_utils.py [hunk 14]
```diff
@@ -129,10 +125,10 @@ def test_is_level_reference_series_axis1_error(df):
 # Test _check_label_or_level_ambiguity_df
 # =======================================
 
+
```
Comment (rhshadrach) [inline pandas/tests/generic/test_label_or_level_utils.py:128]: Here

File: pandas/tests/indexes/datetimes/test_setops.py [hunk 15]
```diff
@@ -226,7 +225,7 @@ def test_intersection(self, tz, sort):
         rng4 = date_range("7/1/2000", "7/31/2000", freq="D", name="idx")
         expected4 = DatetimeIndex([], freq="D", name="idx")
 
-        for (rng, expected) in [
+        for rng, expected in [
```
Comment (rhshadrach) [inline pandas/tests/indexes/datetimes/test_setops.py:228]: Here

File: pandas/tests/indexes/datetimes/test_setops.py [hunk 16]
```diff
@@ -257,7 +256,7 @@ def test_intersection(self, tz, sort):
         expected4 = DatetimeIndex([], tz=tz, name="idx")
         assert expected4.freq is None
 
-        for (rng, expected) in [
+        for rng, expected in [
```
Comment (rhshadrach) [inline pandas/tests/indexes/datetimes/test_setops.py:259]: Here

File: pandas/tests/indexes/period/test_setops.py [hunk 17]
```diff
@@ -193,7 +192,7 @@ def test_intersection_cases(self, sort):
         rng4 = period_range("7/1/2000", "7/31/2000", freq="D", name="idx")
         expected4 = PeriodIndex([], name="idx", freq="D")
 
-        for (rng, expected) in [
+        for rng, expected in [
```
Comment (rhshadrach) [inline pandas/tests/indexes/period/test_setops.py:195]: Here

File: pandas/tests/indexes/period/test_setops.py [hunk 18]
```diff
@@ -227,7 +226,7 @@ def test_intersection_cases(self, sort):
         rng4 = period_range("7/1/2000", "7/31/2000", freq="D", name="idx")
         expected4 = PeriodIndex([], freq="D", name="idx")
 
-        for (rng, expected) in [
+        for rng, expected in [
```
Comment (rhshadrach) [inline pandas/tests/indexes/period/test_setops.py:229]: Here

File: pandas/tests/indexes/timedeltas/test_scalar_compat.py [hunk 19]
```diff
@@ -103,7 +103,7 @@ def test_round(self):
         t1c = TimedeltaIndex([1, 1, 1], unit="D")
 
         # note that negative times round DOWN! so don't give whole numbers
-        for (freq, s1, s2) in [
+        for freq, s1, s2 in [
```
Comment (rhshadrach) [inline pandas/tests/indexes/timedeltas/test_scalar_compat.py:106]: Here

File: pandas/tests/window/test_expanding.py [hunk 20]
```diff
@@ -186,7 +185,7 @@ def test_iter_expanding_dataframe(df, expected, min_periods):
     # GH 11704
     expected = [DataFrame(values, index=index) for (values, index) in expected]
 
-    for (expected, actual) in zip(expected, df.expanding(min_periods)):
+    for expected, actual in zip(expected, df.expanding(min_periods)):
```
Comment (rhshadrach) [inline pandas/tests/window/test_expanding.py:188]: Here

File: pandas/tests/window/test_expanding.py [hunk 21]
```diff
@@ -205,7 +204,7 @@ def test_iter_expanding_series(ser, expected, min_periods):
     # GH 11704
     expected = [Series(values, index=index) for (values, index) in expected]
 
-    for (expected, actual) in zip(expected, ser.expanding(min_periods)):
+    for expected, actual in zip(expected, ser.expanding(min_periods)):
```
Comment (rhshadrach) [inline pandas/tests/window/test_expanding.py:207]: Here

File: pandas/tests/window/test_rolling.py [hunk 22]
```diff
@@ -798,9 +796,7 @@ def test_iter_rolling_dataframe(df, expected, window, min_periods):
     # GH 11704
     expected = [DataFrame(values, index=index) for (values, index) in expected]
 
-    for (expected, actual) in zip(
-        expected, df.rolling(window, min_periods=min_periods)
-    ):
+    for expected, actual in zip(expected, df.rolling(window, min_periods=min_periods)):
```
Comment (rhshadrach) [inline pandas/tests/window/test_rolling.py:799]: Here

File: pandas/tests/window/test_rolling.py [hunk 23]
```diff
@@ -896,9 +892,7 @@ def test_iter_rolling_series(ser, expected, window, min_periods):
     # GH 11704
     expected = [Series(values, index=index) for (values, index) in expected]
 
-    for (expected, actual) in zip(
-        expected, ser.rolling(window, min_periods=min_periods)
-    ):
+    for expected, actual in zip(expected, ser.rolling(window, min_periods=min_periods)):
```
Comment (rhshadrach) [inline pandas/tests/window/test_rolling.py:895]: Here

File: pandas/tests/window/test_rolling.py [hunk 24]
```diff
@@ -948,7 +942,7 @@ def test_iter_rolling_datetime(expected, expected_index, window):
         Series(values, index=idx) for (values, idx) in zip(expected, expected_index)
     ]
 
-    for (expected, actual) in zip(expected, ser.rolling(window)):
+    for expected, actual in zip(expected, ser.rolling(window)):
```
Comment (rhshadrach) [inline pandas/tests/window/test_rolling.py:945]: Here

File: setup.py [hunk 25]
```diff
@@ -393,6 +393,7 @@ def run(self):
 # ----------------------------------------------------------------------
 # Specification of Dependencies
 
+
```
Comment (rhshadrach) [inline setup.py:396]: Here

**Review by rhshadrach — COMMENTED** [review]
Highlighting changes other than removed new lines and the change to black versions

The added newlines are interesting; not sure why it's doing that.

File: setup.py (hunk 25 above)
Comment (MarcoGorelli) [reply setup.py:396]: thanks! did you do this manually or do you have some magical `git diff` option? 🧙

**Comment by MarcoGorelli** [comment]
thanks @rhshadrach for going through the changes - I think they look fine

@lithomas1 is it OK to go ahead with this, or does it need to wait til after the release?

File: setup.py (hunk 25 above)
Comment (rhshadrach) [reply setup.py:396]: Ah, just manual.

**Comment by lithomas1** [comment]
+1 on landing, let's do this before the branch cut.

**Comment by MarcoGorelli** [comment]
cool, I've updated - let's merge on green-ish then?

**Review by lithomas1 — APPROVED** [review]

**Comment by mroeschke** [comment]
Sorry I just merged a few PRs since this got updated.

Since it's a style change that could impact all PRs, lets rebase once more (I won't merge any other PRs) and get this in

**Comment by MarcoGorelli** [comment]
I just tried fetching and merging upstream/main locally, and then did `pre-commit run --all-files`, and everything passed

So I think we should be safe to just merge directly

**Comment by mroeschke** [comment]
Ah nice thanks for checking.

**Review by mroeschke — APPROVED** [review]

**Comment by mroeschke** [comment]
Thanks @MarcoGorelli

**Comment by MarcoGorelli** [comment]
thank you

> So I think we should be safe to just merge directly

If this doesn't work I accept full blame

**Pull request merged**